modules/router/prisma-postgres/schema.prisma

```diff
@@ -17,3 +17,33 @@ model QueuedUpdate {
   status         String
   context        String?
 }
+
+model AutoRebalance {
+  id                      String  @id @default(uuid())
+  status                  String
+  approveHash             String?
+  executeHash             String?
+  completeHash            String?
+  
+  // Swap info
+  fromChainId             String
+  fromAssetId             String
+  toChainId               String
+  toAssetId               String
+  priceType               String
+  hardcodedRate           String
+  rebalancerUrl           String?
+  rebalanceThresholdPct   Int?
+  percentageFee           Int?
+  flatFee                 String?
+  gasSubsidyPercentage    Int?
+
+
+  createdAt               DateTime @default(now())
+  updatedAt               DateTime @default(now())
+
+  @@unique([fromChainId, toChainId, fromAssetId, toAssetId])
+  @@unique(approveHash)
+  @@unique(completeHash)
```

Comment:
I think we'll need to remove these unique constraints. Below you mention that the completion tx hash won't be present for the L1->L2 rebalance. If that's the case it will violate the constraint. I think it's acceptable to loosen the restriction and remove the uniques on the hashes. I appreciate the specificity though 👍.

Reply:
Should we add `@@unique()` for ID column above? Wondering why that's not a thing.

Reply:
`id` columns are unique by default since they are primary key.